main.go

```diff
@@ -43,13 +43,16 @@ func main() {
 	}
 	logger.L().Debug("loaded config for components", helpers.Interface("components", components))
 
-	services, err := config.GetServiceURLs("/etc/config/services.json")
-	if err != nil {
-		logger.L().Ctx(ctx).Fatal("failed discovering urls", helpers.Error(err))
-	}
+	var eventReceiverRestURL string
+	if components.Components.ServiceDiscovery.Enabled {
+		services, err := config.GetServiceURLs("/etc/config/services.json")
+		if err != nil {
+			logger.L().Ctx(ctx).Fatal("failed discovering urls", helpers.Error(err))
+		}
 
-	eventReceiverRestURL := services.GetReportReceiverHttpUrl()
-	logger.L().Debug("setting eventReceiverRestURL", helpers.String("url", eventReceiverRestURL))
+		eventReceiverRestURL := services.GetReportReceiverHttpUrl()
+		logger.L().Debug("setting eventReceiverRestURL", helpers.String("url", eventReceiverRestURL))
+	}
 
 	cfg, err := config.LoadConfig("/etc/config")
```

Comment:
This is confusing, the name of the file is the same name as the directory: `config.GetServiceURLs("/etc/config/services.json")`

Reply:
but this is how viper works... you don't specify the filename, but only the dir